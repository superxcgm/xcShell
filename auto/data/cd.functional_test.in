mkdir cd_dir
cd cd_dir
cd
cd -
cd ~
cd -
cd ..
cd .