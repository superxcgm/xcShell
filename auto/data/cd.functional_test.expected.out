xcShell_functional_test > mkdir cd_dir
xcShell_functional_test > cd cd_dir
cd_dir > cd
~ > cd -
cd_dir > cd ~
~ > cd -
cd_dir > cd ..
xcShell_functional_test > cd .
xcShell_functional_test > cd /not-exist
No such file or directory
xcShell_functional_test >